src/main/resources/public/partials/header/header.js

```diff
@@ -16,15 +17,23 @@ angular.module('learny').directive(
                                 serverCommunicator.logoutAsync().success(
                                         function(data, status, headers, config) {
                                             console.log('Successfully logged out');
-                                            $state.go('login', {}, {
-                                                reload : true
-                                            });
+                                            $state.go('welcome');
                                         }).error(function(data, status, headers, config) {
                                     console.log('Logout failed');
                                 });
 
                             }
 
+                            $scope.showlogin = false;
```

Comment:
Der header enthält jetzt die Loginform, die nur bei einem Click auf den Login Button angezeigt wird. Beim anzeigen wird geprüft ob der Nutzer schon angemeldet ist, damit er ggf. direkt in die Anwendung geleitet werden kann